POCLibrary/POCLibrary/Class1.cs

```diff
@@ -11,6 +11,7 @@ public class Class1
         public Class1()
         {
             var someUnusedVar = 0;
+            var someUnusedVar2 = 0;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused "someUnusedVar2" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=csharpsquid%3AS1481)